pkgs/misc/mosquitto-go-auth/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub, pkgs }:
+
+buildGoPackage rec {
```

Comment:
They seem to have `go.mod` and `go.sum` files in their repo, did you try to build it with `buildGoModule`?